dotcom-rendering/src/lib/ad-targeting.test.ts

```diff
@@ -1,63 +1,42 @@
-import { Article } from '@root/fixtures/generated/articles/Article';
 import { buildAdTargeting } from './ad-targeting';
 
-const CAPI = {
-	...Article,
-	config: {
-		...Article.config,
-		adUnit: '/59666047/theguardian.com/money/article/ng',
-		edition: 'UK',
-		section: 'money',
-		sharedAdTargeting: {
-			ct: 'article',
-			co: ['rob-davies'],
-			url: '/money/2017/mar/10/ministers-to-criminalise-use-of-ticket-tout-harvesting-software',
-			su: ['0'],
-			edition: 'uk',
-			tn: ['news'],
-			p: 'ng',
-			k: [
-				'ticket-prices',
-				'consumer-affairs',
-				'internet',
-				'viagogo',
-				'money',
-			],
-			sh: 'https://theguardian.com/p/64ak8',
-		},
-	},
+const sharedAdTargeting = {
+	co: ['rob-davies'],
+	ct: 'article',
+	edition: 'uk',
+	k: ['ticket-prices', 'consumer-affairs', 'internet', 'viagogo', 'money'],
+	p: 'ng',
+	sh: 'https://theguardian.com/p/64ak8',
+	su: ['0'],
+	tn: ['news'],
+	url: '/money/2017/mar/10/ministers-to-criminalise-use-of-ticket-tout-harvesting-software',
 };
 
 describe('buildAdTargeting', () => {
 	const expectedAdTargeting = {
 		adUnit: '/59666047/theguardian.com/money/article/ng',
 		customParams: {
-			cc: 'UK',
-			co: ['rob-davies'],
-			ct: 'article',
-			edition: 'uk',
-			inskin: 'f',
-			k: [
-				'ticket-prices',
-				'consumer-affairs',
-				'internet',
-				'viagogo',
-				'money',
-			],
-			p: 'ng',
-			pa: 'f',
-			s: 'money',
-			sh: 'https://theguardian.com/p/64ak8',
-			su: ['0'],
-			tn: ['news'],
-			url: '/money/2017/mar/10/ministers-to-criminalise-use-of-ticket-tout-harvesting-software',
 			sens: 'f',
 			si: 'f',
 			vl: 0,
+			cc: 'UK',
```

Comment:
'UK' is not a valid country code. 'GB' is.

Reply:
It looks from the [CAPI documentation](https://open-platform.theguardian.com/documentation/edition) like `edition`s / edition `id`s are not ISO country codes.
The response from the [/editions endpoint](https://content.guardianapis.com//editions?api-key=test) suggests that any of ('UK', 'US', 'AU', 'International') would be valid values.

Reply:
Oh wow… [`cc` in AdManager][cc] does not have predefined value (dynamic) and isn’t used in reporting, so I guess we’re fine? Ideally, we’d pass in the real country code. Worth checking whether it’s not a deprecated targeting, too.

[cc]: https://admanager.google.com/59666047#inventory/custom_targeting/detail/custom_key_id=11703293

Reply:
Good call, I hadn't seen the lack of targeting values in GAM! It does look like `cc` may no longer be in use.